### PR TITLE
feat: Implement switching to next/previous block on detail pages

### LINKED ISSUE #375: Implement switching to next/previous block on detail pages
### Please describe the problem that your request should resolve.
Implement navigation to previous/next keyblock and microblock on the respective details pages

### LINKED ISSUE #406: Microblocks tab is visible when the keyblock is not mined
### Context information

application version: 
device: 
browser: 
operating system: 

### Steps to reproduce
- Open a keyblock which is not mined

### What is expected?
The microblocks tab should be hidden.

### What is actually happening?

image

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #375 
fixes #406

compared to the visual material, the design is old for consistency and will be improved here https://github.com/aeternity/aescan/issues/148

## Demo

https://github.com/aeternity/aescan/assets/15363559/ad317fd1-d667-4db6-8e55-e6f538238081

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add navigation through keyblock detail

### CODE REVIEW

**Review by michele-franchi — COMMENTED** [review]
It looks good but is there a way to disable the next arrow if the keyblock is the last one to avoid the "Requested keyblock has never been seen in the network." screen?

**Comment by lukeromanowicz** [comment]
I think it should be doable by checking height against blockHeight in recentBlocksStore

**Comment by janmichek** [comment]
> It looks good but is there a way to disable the next arrow if the keyblock is the last one to avoid the "Requested keyblock has never been seen in the network." screen?

Fixed and regenerated the demo

File: src/components/KeyblockDetailsPanel.vue [hunk 1]
```diff
@@ -28,7 +28,32 @@
...
+                </button>
+              </app-link>
+              {{ keyblockDetails.height }}
+              <app-link :to="`/keyblocks/${keyblockDetails.height + 1}`">
+                <button
+                  :class="[
+                    'keyblock-details-panel__button',
+                    'keyblock-details-panel__button--next',
+                  ]"
+                  :disabled="!isNextKeyblockMined">
```
Comment (lukeromanowicz) [inline src/components/KeyblockDetailsPanel.vue:50]: the button is still clickable, even if it's disabled :/

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
I expected that more recent keyblock button will be on the left side since every content we display is always latest first, and older stuff is available when you navigate right. But I guess this works too, just a little bit unexpected.

Overall very good job, I left only one not but I'm afraid it needs to be fixed before merging

File: src/components/KeyblockDetailsPanel.vue (hunk 1 above)
Comment (janmichek) [reply src/components/KeyblockDetailsPanel.vue:52]: Good catch. You are right, haven't spotted this. Fixed

**Review by lukeromanowicz — APPROVED** [review]
Now it works great, thank you for the fix!

**Review by michele-franchi — APPROVED** [review]
It works great!

Could you please open a followup to improve arrows handling and enable again the next one if a new keyblock comes from the websocket? Not super needed right now but this would improve it.

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**